contracts/token/ERC721/ERC721.sol

```diff
@@ -423,8 +423,8 @@ contract ERC721 is Context, ERC165, IERC721, IERC721Metadata {
         address to,
         uint256 tokenId
     ) internal virtual {}
-
-    /**
+     
+     /**
```

Comment:
What IDE are you using? Can you configure it to remove trailing whitespaces when saving?
this part of the commit is really strange

Reply:
I'm using VScode, just turned on the setting, the trailing space should not be there now.